WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift

```diff
@@ -97,7 +98,8 @@ open class QuickStartTourGuide: NSObject {
                             message: tour.description,
                             style: noticeStyle,
                             actionTitle: tour.suggestionYesText,
-                            cancelTitle: tour.suggestionNoText) { [weak self] accepted in
+                            cancelTitle: tour.suggestionNoText,
+                            tag: identifier) { [weak self] accepted in
```

Comment:
There is another notice that is being dispatched in `showStepNotice`. That one should be using the `Notice.Tag` as well. 🙂 Right now, when I perform these steps, the notice is not dismissed:

1. Create a WordPress.com site and accept to enable Quick Start when asked.
2. In the Site Page, tap on _Customize Your Site_ → Upload a site icon. A notice will be shown. 
3. Tap Back. 

The notice never disappears. It stays like this forever: 

<img width="320" alt="iPhone Xs — 12 2 2019-11-04 16-10-54" src="https://user-images.githubusercontent.com/198826/68165839-c83baf80-ff1d-11e9-8b64-265f885d5aa1.png">